domains/roman.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "GameRoMan",
+    "email": "gameromandev@gmail.com"
+  },
+  "record": {
+    "CNAME": "gameroman.pages.dev"
```

Comment:
Seems to have little content.

Reply:
Can confirm, there were games; I enjoyed your Tetris game haha